src/createServer.js

```diff
@@ -1,3 +1,58 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const requestUrl = new URL(req.url, `http://${req.headers.host}`);
+    const text = requestUrl.pathname.substring(1);
```

Comment:
Consider adding a check to ensure that `text` is not an empty string after `substring(1)`. If the URL path is just '/', `text` will be an empty string, which might not be the intended behavior.